ftcosi/protocol/protocol_test.go

```diff
@@ -99,27 +100,24 @@ func TestProtocol(t *testing.T) {
 	}
 }
 
-// Tests unresponsive leaves in various tree configurations
-func TestUnresponsiveLeafs(t *testing.T) {
-	nodes := []int{3, 13, 24}
-	subtrees := []int{1, 2}
+// Tests various trees configurations
+func TestProtocolQuickAnswer(t *testing.T) {
```

Comment:
how do we know these are actually the quick answers, is it possible to check how long it takes for the signature to arrive?

Reply:
I count the bits in the signature and make sure it's less than the total number of nodes - that should hold up most of the time, except with very few nodes and a very fast server.